sherpa/conftest.py

```diff
@@ -38,11 +38,13 @@
 
 
 TEST_DATA_OPTION = "--test-data"
-
+SHOW_WARNINGS_OPTION = "--show-warnings"
 
 def pytest_addoption(parser):
     parser.addoption("-D", TEST_DATA_OPTION, action="store",
                      help="Alternative location of test data files")
+    parser.addoption(SHOW_WARNINGS_OPTION, action="store_true",
```

Comment:
Ah, the problem is that I wasn't passing 'sherpa' as an argument. I am not sure if the 'test' command should be implemented differently, in any case the following command both work as expected:

````
python setup.py test -a "-kds9 sherpa"
python setup.py test -a "-kds9 --show-warnings sherpa"
````